js/adapt-mathJax.js

```diff
@@ -1,20 +1,50 @@
 define([ "core/js/adapt" ], function(Adapt) {
 
+	function loadScript(scriptObject, callback) {
+		var head = document.getElementsByTagName('head')[0];
+		var script = document.createElement('script');
+
+		script.type = scriptObject.type || 'text/javascript';
+    
+		if (scriptObject.src) {
+			script.src = scriptObject.src;
+		}
+
+		if (scriptObject.text) {
+			script.text = scriptObject.text;
+		}
+    
+		if (callback) {
+			// Then bind the event to the callback function.
+			// There are several events for cross browser compatibility.
+			script.onreadystatechange = callback;
+			script.onload = callback;
+		}
+    
+        // Append the <script> tag.
+        head.appendChild(script);
+	}
+
 	function setUpMathJax() {
+		Adapt.trigger('plugin:beginWait');
+
 		var config = Adapt.config.get("_mathJax");
 		var inlineConfig = config ? config._inlineConfig : {
 				"extensions": [ "tex2jax.js" ],
 				"jax": [ "input/TeX", "output/HTML-CSS" ]
 		};
-		var src = config ?
-			config._src :
-			"//cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.2/MathJax.js";
-		var $init = $("<script/>", { src: "assets/mathJaxInit.js" });
-		var $config = $("<script/>", { type: "text/x-mathjax-config" });
-		var $src = $("<script/>", { src: src });
-
-		$config[0].text = "MathJax.Hub.Config(" + JSON.stringify(inlineConfig) + ");";
-		$("head").append($init, $config, $src);
+		var src = config ? config._src : "//cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.2/MathJax.js";
+
+		loadScript({ 

```

Comment:
sorry if this is a bit of a daft question but could `$.getScript()` not handle this? If not, no need to explain why - just say 'no'!